portal-api/src/v1/gef/models/facilities.js

```diff
@@ -64,6 +68,10 @@ class Facility {
         this.hasBeenIssued = Boolean(req.hasBeenIssued);
       }
 
+      if (req.hasBeenIssuedAndAcknowledged != null) {
```

Comment:
if we expect this to be `true`, then can we use 
`if (req.hasBeenIssuedAndAcknowledged) {
...
}`

Reply:
done

Reply:
I can't see the changes in here?